src/angularComponent/ngcOmniboxController.js

```diff
@@ -64,9 +64,12 @@ export default class NgcOmniboxController {
     }, true);
 
     // Remove the focus ring when the overall component is focused
-    const styleSheets = this.doc.styleSheets;
-    if (styleSheets && styleSheets.length) {
-      styleSheets[styleSheets.length - 1].insertRule('ngc-omnibox:focus {outline: none}', 0);
+    const head = this.doc.head;
+    if (head) {
```

Comment:
What happens if there is no head element?

Reply:
If there's no head it will just not apply that rule, similar to what was happening when document.styleSheets did not exist. But every webpage should have a head, so I don't think this case will happen very often

Reply:
Should you give the style element an ID, and check if it already exists? This will prevent this element from being added repetitively by multiple instances of the component.

Reply:
> Should you give the style element an ID, and check if it already exists? This will prevent this element from being added repetitively by multiple instances of the component.

Done